src/pages/scripting/samples/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Samples
+description: UXP Script samples
+contributors:
+  - https://github.com/amandahuarng
+---
+
+# Samples
+
+## Create a new layer 
+You can create a script that creates a new layer, with the help of the Photoshop DOM API: 
+```js
+const app = require('photoshop').app;
+await app.documents.add();
+await app.activeDocument.createLayer();
+app.activeDocument.layers[0].name = 'New layer';
+```
+
+## Access the local filesystem
+You can create a script that accesses your local filesystem, with the help of the UXP storage module: 
+```js
+const uxpfs = require("uxp").storage;
+const ufs = uxpfs.localFileSystem;
+
+await ufs.getTemporaryFolder().
+    then((result) => {
+        return result.getMetadata();
+    }).then((result) => {
+        console.log(JSON.stringify(result));
+    }).catch((error) => {
+        console.log("Local File system Error: " + error);
```

Comment:
Wouldn't be easier for understanding to use async/await here instead of then/catch?

Reply:
@jardicc are you suggesting something like this?
```
const uxpfs = require("uxp").storage;
const ufs = uxpfs.localFileSystem;

try {
    const folder = await ufs.getTemporaryFolder()
    const metadata = await folder.getMetadata();
    console.log(JSON.stringify(metadata));
} catch (e) {
    console.log(`Local File system error: ${e}`);
}
```

Reply:
Yes exactly. I think this is easier to understand and read for beginners.